packages/components/date-picker/src/date-picker-com/panel-date-pick.vue

```diff
@@ -762,6 +764,13 @@ watch(
   { immediate: true }
 )
 
+// When close time-pick-panel, blur the timePickerInput
+watch(timePickerVisible, (newValue) => {
+  if (!newValue) {
+    timePickerInputRef.value?.blur()
+  }
+})
+
```

Comment:
Perhaps it would be better to execute `timePickerInputRef.value?.blur()` inside the `handleTimePickClose` function.